.github/CONTRIBUTING.md

```diff
@@ -56,7 +63,7 @@ various providers, whether it's the primary Instill Model or those from
 third-party AI vendors. They are defined and initialized in the [`ai`](../ai)
 package.
 
-#### Data
+#### `Data`
```

Comment:
❓ Do we want this headers with `code` style? Generic, AI, Data, etc. are end-user names, not IDs, and they just appeared as **bold** in the parent section.

Reply:
I was thinking to stand out those component types from the parent header. If they look weird and breaking the header hierarchy, I can reset it.

Reply:
To me it was a bit confusing, headers already convey emphasis so it lead me to think the code style meant something else.